.github/CONTRIBUTING.md

```diff
@@ -95,7 +95,7 @@ The previous code made compliant:
 /datum/datum1/datum2/proc/proc3()
 	code
 /datum/datum1/datum2/proc2()
-	..()
+	. = ..()
```

Comment:
This shouldn't be here. I'm putting it down to muscle memory

Reply:
This is included in the body of the PR.

Reply:
The point of this block of code is to demonstrate relative vs absolute pathing, don't update one without updating the other. I'm also not sure . = ..() should be here, but people will need to ask about it at some point.

Reply:
We ask contributors to use it instead of `..()` unless they have a specific reason not to. It would make sense for the contribution guidelines to do the same.